stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -582,14 +635,14 @@ julia> norm(v, 1)
 julia> norm(v, Inf)
 6.0
 
-julia> norm([1 2 3; 4 5 6; 7 8 9])
-16.881943016134134
+julia> norm([1 2 3; 4 5 6; 7 8 9]) ≈ sqrt(sum(abs2, 1:9))
+true
 
-julia> norm([1 2 3 4 5 6 7 8 9])
-16.881943016134134
+julia> norm([1 2 3 4 5 6 7 8 9]) ≈ sqrt(sum(abs2, 1:9))
+true
 
-julia> norm(1:9)
-16.881943016134134
```

Comment:
Was off by one epsilon. This is expected. Reason is that somewhere we optimized `a/b` into `binv = inv(b); a * binv`. Also, SIMD is used now (not in this specific example), which can also give slightly different results.